device-frames/frames-config.js

```diff
@@ -0,0 +1,9 @@
+module.exports.framesConfig = {
```

Comment:
```suggestion
module.exports = {
```

And the filename should be `config.js`.